doc/ChangeLog

```diff
@@ -1,5 +1,55 @@
 ===============================================================
 
+Tag name: atmos_phys0_0x_xxx
```

Comment:
Just FYI that the tag will likely be `atmos_phys0_04_000`.

Reply:
Thx. Updated.